src/CONST.js

```diff
@@ -32,6 +32,11 @@ const CONST = {
     // Minimum width and height size in px for a selected image
     AVATAR_MIN_WIDTH_PX: 80,
     AVATAR_MIN_HEIGHT_PX: 80,
+
+    // Maximum width and height size in px for a selected image
+    AVATAR_MAX_WIDTH_PX: 2048,
```

Comment:
We are having issues when avatar height or width is greater than around 4500/4800 pixels.

Facebook uses the following resolution:

> FB Avatar size:
Max Size: 2048px X 2048px
Min Size: 168px X 168px
Aspect ratio: 1:1

So, max size is set 2048px here. @tgolen Are we okay with this or want to increase this size?

Reply:
While I’m also testing various resolutions on my end, could you help me with testing e.g. 4096 on different platforms including real mobile device ? @sobitneupane

Reply:
> We are having issues when avatar height or width is greater than around 4500/4800 pixels.

@sobitneupane could you please expand on what kind of issues we have seen?

If I'm not wrong, looks like 2048 is coming from [this](https://promo.com/blog/social-media-image-sizes) reference?
And, comparing to other platforms, it seems more than enough. I would be down to decrease it, but not increase it.

@shawnborton can we please have your thoughts in this case

Reply:
> > We are having issues when avatar height or width is greater than around 4500/4800 pixels.
> 
> @sobitneupane could you please expand on what kind of issues we have seen?

This [issue](https://github.com/Expensify/App/issues/13006) was reported for images larger than 5000 pixels. (https://expensify.slack.com/archives/C049HHMV9SM/p1669089889220939)
> If I'm not wrong, looks like 2048 is coming from [this](https://promo.com/blog/social-media-image-sizes) reference? And, comparing to other platforms, it seems more than enough. I would be down to decrease it, but not increase it.

With 2048 px AVATAR_MAX_WIDTH_PX, users might face difficulty uploading profile avatar with Take photo option (Upload photo> Take photo) in native because image resolution might be greater than 2048px.


https://user-images.githubusercontent.com/25876548/210228000-1bd89b98-cf84-4153-b980-f126336ab9a7.MP4

Reply:
Thank you, got it!
I did some testing on Whatsapp, and looks like it compresses the image as per their accepted criteria.
I feel like we should do the same, and not ask user to compress the image according to our standards.
If we increase the limit for now to allow photos captured directly through camera, it doesn't really fix the issue, no? or it will create a new issue - "Users on _this device_ can't upload profile picture directly through camera"?

Reply:
Just chiming in to say I agree with Monil's line of thought above. 2048 seems fine to me assuming we don't accidentally block things like camera uploads.

Reply:
Why did we choose 2048? Why wouldn't we just choose the max size until we know there can be an issue with some native devices (eg. 4400)? I think that before we consider pre-compression, we should understand what the resolution is of native camera uploads first.

Reply:
FYI - By default my iPhone 13 Pro takes `3024 × 4032` so I agree `2048` is too small to block on.

Reply:
I can:
- Increase max resolution to: 4096px
- Set max resolution for camera upload to: 4096px

https://github.com/react-native-image-picker/react-native-image-picker#options

Reply:
Oh, cool. I was wondering if we could specify a max for the camera.